packages/modeling/src/geometries/geom2/type.d.ts

```diff
@@ -0,0 +1,9 @@
+import Vec2 from '../../maths/vec2/type'
+import Mat4 from '../../maths/mat4/type'
+
+export interface Geom2 {
+  sides: Array<[Vec2, Vec2]>
+  transforms?: Mat4
```

Comment:
@ahdinosaur any ideas why transforms is declared as 'optional'?

Reply:
oops, that's my bad, i misread the `create` function!